### PR TITLE
Upgrade webpack-dev-server and webpack-cli dependencies

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
https://issues.redhat.com/browse/RHSTOR-6588

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -179,14 +179,13 @@
...
     "stylelint-config-standard": "^22.0.0",
     "stylelint-order": "^4.1.0",
-    "webpack-dev-server": "^4.15.2"
+    "webpack-dev-server": "^5.1.0"
   },
   "resolutions": {
     "jest-environment-jsdom/jsdom": "^20.0.3",
     "webpack": "5.94.0",
     "webpack-bundle-analyzer/ws": "^7.5.10",
     "webpack-dev-server/express": "^4.21.0",
```
Comment (SanjalKatiyar) [inline package.json:188]: latest `webpack-dev-server` still using `"express": "^4.19.2",`

Comment (SanjalKatiyar) [reply package.json:188]: https://www.npmjs.com/package/webpack-dev-server?activeTab=code

File: package.json [hunk 2]
```diff
@@ -126,7 +126,7 @@
     "victory-core": "^35.4.4",
     "webpack": "5.94.0",
     "webpack-bundle-analyzer": "^4.10.2",
-    "webpack-cli": "4.5.x",
+    "webpack-cli": "5.1.4",
```
Comment (alfonsomthd) [inline package.json:129]: @SanjalKatiyar provided we have a `yarn.lock` IMHO it's harmless to add the caret (`^5.1.4`) so when we run in the future `yarn upgrade` this package also get upgraded (same for `webpack` package).

**Pull request merged**